src/components/Post.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import styled from 'styled-components'
+
+
+
+// eslint-disable-next-line import/no-anonymous-default-export
+export default ({ post }) => {
+
+
+
+	return (
+
+		<S.WraperCard >
+			<img src={post.image.url} alt="..."></img>
+			<h3 >{post.name}</h3>
+		</S.WraperCard>
+	)
+}
+
+
+const S = {
+	WraperCard: styled.div`
```

Comment:
Try avoid using stings.
Use like ...: styled.div({
 <your styles here>
});
And replace all styles to additional file that would be responsible for storing styles.3